CompilerSource/compiler/compile.cpp

```diff
@@ -700,7 +703,7 @@ wto << "namespace enigma_user {\nstring shader_get_name(int i) {\n switch (i) {\
   fclose(gameModule);
 
   // Run the game if requested
-  if (mode == emode_run or mode == emode_debug or mode == emode_design)
+  if (run_game && (mode == emode_run or mode == emode_debug or mode == emode_design))
```

Comment:
@JoshDreamland does this work for now? I have to keep all of the mode checks or else LGM will start running the game in compile mode, which it shouldn't.

Reply:
This is how I expected this line to look, if that's what you're asking.